packages/core/src/App/Containers/RealAccountSignup/add-crypto-currency.jsx

```diff
@@ -4,13 +4,14 @@ import React from 'react';
 import { FormSubmitButton } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import { isMobile } from '@deriv/shared';
 import { Hr, RadioButtonGroup, RadioButton } from './currency-selector.jsx';
 
 const messages = [
-    'Choose your preferred cryptocurrency',
-    'You can open an account for each cryptocurrency.',
-    'Add a real account',
-    'Choose a currency you would like to trade with.',
+    localize('Add a cryptocurrency account'),
```

Comment:
if we are using localize function instead of component, `messages` needs to be a function and can't be a constant, otherwise localize function will just return english.